server/engine/tests/test_acc_minimize_pass.py

```diff
@@ -0,0 +1,26 @@
+import nimblephysics as nimble
+import unittest
+import os
+from inspect import getsourcefile
+import shutil
+from passes.acc_minimize_pass import add_acc_minimize_pass
+import tempfile
+
+TESTS_PATH = os.path.dirname(getsourcefile(lambda:0))
+TEST_DATA_PATH = os.path.join(TESTS_PATH, 'data')
+
+
+class TestAccMinimizePass(unittest.TestCase):
```

Comment:
Since acceleration minimization, the dynamics pass, and missing grf detection all fit under the "dynamics fitting" pipeline, you could probably combine these three unittests into the same `test_*.py` file. Reduces file clutter a bit, but I think it is also helpful to group these test together since they are inherently related.

This might imply that the acceleration minimization and missing grf detection could possibly be absorbed within the "dynamics pass" step, but as of writing this comment I haven't reviewed those sections to know if that coupling would make sense.

Reply:
I've rearranged that way. I can tell from this comment and a few other ones that that makes more sense in your mental model, so I'm happy to move things over that way. I was maybe imagining an excess of unnecessary abstraction, that you could use these pieces separately if you wanted (though why would you?). So I've grouped both the code and the tests under the `dynamics_fitter` conceptual header.